index.html

```diff
@@ -3,9 +3,270 @@
+    <section class="mainProgram" id="mainProgram">
+      <h2>Main Program</h2>
+      <hr class="redLine" />
+
+      <div class="programContainer">
+        <!-- program1 -->
+        <div class="programs">
+          <img src="images/lecture.png" alt="Exhibition" class="programIcon" />
+          <h4 class="programHeading">Lecture</h4>
+          <p class="programPara">
+            Esteemed panelists and experts lead insightful sessions exploring
+            cricket's global impact, fostering dialogue on addressing the
+            challenges and opportunities facing the sport.
+          </p>
+        </div>
+
+        <!-- program2 -->
+        <div class="programs">
+          <img
+            src="images/exhibition.png"
+            alt="Exhibition"
+            class="programIcon"
+          />
+          <h4 class="programHeading">Exhibition</h4>
+          <p class="programPara">
+            Esteemed panelists and experts lead insightful sessions exploring
+            cricket's global impact, fostering dialogue on addressing the
+            challenges and opportunities facing the sport.
+          </p>
+        </div>
+
+        <!-- program3 -->
+        <div class="programs">
+          <img src="images/forum.png" alt="Exhibition" class="programIcon" />
+          <h4 class="programHeading">Forum</h4>
+          <p class="programPara">
+            Esteemed panelists and experts lead insightful sessions exploring
+            cricket's global impact, fostering dialogue on addressing the
+            challenges and opportunities facing the sport.
+          </p>
+        </div>
+
+        <!-- program4 -->
+        <div class="programs">
+          <img src="images/workshop.png" alt="Exhibition" class="programIcon" />
+          <h4 class="programHeading">Workshop</h4>
+          <p class="programPara">
+            Esteemed panelists and experts lead insightful sessions exploring
+            cricket's global impact, fostering dialogue on addressing the
+            challenges and opportunities facing the sport.
+          </p>
+        </div>
+
+        <!-- program5 -->
+        <div class="programs">
+          <img src="images/ignite.png" alt="Exhibition" class="programIcon" />
+          <h4 class="programHeading">Ignite</h4>
+          <p class="programPara">
+            Esteemed panelists and experts lead insightful sessions exploring
+            cricket's global impact, fostering dialogue on addressing the
+            challenges and opportunities facing the sport.
+          </p>
+        </div>
+
+        <a href="#" class="mobileJoin">Join Cricketing Fusion</a>
+        
+      </div>
+      <a href="#" class="desktopJoin">Join Cricketing Fusion 2023</a>
+    </section>
+
+    <!-- speakers section -->
+    <section class="speakersContainer">
+      <h2>Featured Speakers</h2>
+      <hr class="redLine" />
+
+      <div class="speakersList">
+        <!-- dynamically created section -->
+      </div>
+
```

Comment:
Excellent work implementing the `More` and `Less` functionality on this button! 💯 Kudos for this! 💯 💯 💯 

- [ ] Let's make some changes here to make your projects look exactly as the design provided. Kindly look at the bottom, it doesn't have a background color. It only has a border, the arrow it's color red, all the letters are in uppercase and the size it's smaller on the Behance design. Also, the size of the button it's almost the entry width of the screen. Please made these changes 💪🏼 

Your design

![image](https://github.com/bhushan354/capstone-conferencePage/assets/93630700/50b1d198-230f-4352-990a-0d4e63f2ce48)

Behance design

![image](https://github.com/bhushan354/capstone-conferencePage/assets/93630700/d7f85238-4e43-4187-96b8-76180bc8b5ac)